code/datums/uplink_item.dm

```diff
@@ -53,23 +56,29 @@ var/list/uplink_items = list()
 	var/abstract = 0
 	var/list/gamemodes = list() // Empty list means it is in all the gamemodes. Otherwise place the gamemode name here.
 	var/list/excludefrom = list() //Empty list does nothing. Place the name of gamemode you don't want this item to be available in here.
-	var/list/job = null
+	var/list/job = list() //Empty list does nothing. If list is not empty, jobs outside this list get 25% extra cost on the item.
 	var/only_on_month	//two-digit month as string
 	var/only_on_day		//two-digit day as string
 	var/num_in_stock = 0	// Number of times this can be bought, globally. 0 is infinite
 	var/static/times_bought = 0
 	var/refundable = FALSE
 	var/refund_amount // specified refund amount in case there needs to be a TC penalty for refunds.
 
+/datum/uplink_item/proc/get_cost(var/user_job, var/cost_modifier = 1)
+	. = cost //"." is our return variable, effectively the same as doing "var/X", working on X, then returning X
+	if(job.len && !matches_job(user_job))
+		. = cost * 1.25
+	. = round(. * cost_modifier)
```

Comment:
round with only one arg works as floor, e.g.:
* 3*1.25 = 3.75 = 3, not 4
* 18*1.25 = 22.5 = 22, not 23 (This is the cost of the banannon)